doc/lineedit.txt

```diff
@@ -1185,7 +1185,7 @@ ifndef::basebackend-html[`@`]
 --
 
 vi-edit-and-accept::
-Start the vi editor to edit the current buffer contents.
+Start VISUAL, EDITOR, or the vi editor to edit the current buffer contents.
```

Comment:
Let's prepend `$` in the monospace font to clarify these are variables.

```suggestion
Start `$VISUAL`, `$EDITOR`, or the vi editor to edit the current buffer contents.
```

Reply:
Other variables in the documentation don't use a `$` before them. Would just using the monospace font be sufficient?